README.md

```diff
@@ -176,6 +177,28 @@ cell.valueOf();
 
 Returns an object for `JSON.stringify()` purpose.
 
+### isEqual(cell)
+
+Determines whether specified cell is deep equal cell.
+
+Parameter | Type            | Description
+----------|-----------------|-----------------------
+`cell`    | `BemCell`       | The cell to compare.
+
+```js
+const BemCell = require('@bem/cell');
+const BemEntityName = require('@bem/entity-name');
+const buttonCell = new BemCell({
+    entity: new BemEntityName({ block: 'button' }), tech: 'css', layer: 'desktop' 
+});
+const inputCell = new BemCelL({
+    entity: new BemEntityName({ block: 'input' }), tech: 'css', layer: 'common' 
+});
+
+buttonCell.isEqual(buttonCell); // true
```

Comment:
maybe it is better to provide an example with comparing to another cell?
not to itself, just to add more clearance.